packages/phalanx/src/Phalanx_KokkosDeviceTypes.hpp

```diff
@@ -16,27 +16,23 @@ namespace PHX {
 
   //#include <Kokkos_Cuda.hpp>
   typedef Kokkos::Cuda Device; 
-  typedef Kokkos::LayoutRight Layout;
   
 #elif defined(PHX_KOKKOS_DEVICE_TYPE_OPENMP)
   
   //#include <Kokkos_hwloc.hpp>
   //#include <Kokkos_OpenMP.hpp>
   typedef Kokkos::OpenMP Device; 
-  typedef Kokkos::LayoutRight Layout;
```

Comment:
I suppose we are getting rid of this entirely. I don't think I have ever used the PHX::Layout type.